File: Codechef/Monopoly/Monopoly.py
T = int(input())
while T:
    T -= 1
    R1, R2, R3 = map(int, input().split())
    if (R1 > (R2 + R3)):
        print("YES")
    elif (R2 > (R1 + R3)):
        print("YES")
    elif (R3 > (R1 + R2)):
        print("YES")
    else:
        print("NO")

File: Codechef/Monopoly/Monopoly.cpp
#include <iostream>
using namespace std;
int main()
{
    int T;
    cin >> T;
    while (T--)
    {
        int R1, R2, R3;
        cin >> R1 >> R2 >> R3;
        if (R1 > (R2 + R3))
        {
            cout << "YES" << endl;
        }
        else if (R2 > (R1 + R3))
        {
            cout << "YES" << endl;
        }
        else if (R3 > (R1 + R2))
        {
            cout << "YES" << endl;
        }
        else
        {
            cout << "NO" << endl;
        }
    }

    return 0;
}